### DISCUSSION TITLE
How exactly do I ensure support for fully utf8 email addresses?

### DISCUSSION THREAD

**Discussion by tomicakorac** [Mailu/Mailu#3631] [User support] [answered]
Disclaimer: This is a fully newbie question, I am only starting to explore the Mailu world, or for that matter the entire email administration.

I am a member of a non-profit organisation tasked to preserve and promote non-latin writing scripts, such as cyrillic. I want to run my own email server using Mailu, in which I want to allow my users to use email addresses entirely in Serbian language, for example `контакт@србија.срб`. My questions are following:

- Is this possible with the latest Mailu version?
  - If yes, how exactly can I achieve it? Right now, if I try to create a new email address with non-ascii characters, I get an error 'invalid value'.
  - If no, is it known what exactly are the reasons?

**Comment by nextgens** [comment, marked as answer]
> what exactly are the reasons?

No one cared enough to make it work. PRs are welcome.
If I were you I would start by contributing tests, identify where things break and then attempt to fix them.
